microprofile-health/src/test/java/org/jboss/eap/qe/microprofile/health/HealthUrlProvider.java

```diff
@@ -0,0 +1,27 @@
+package org.jboss.eap.qe.microprofile.health;
+
+import org.jboss.eap.qe.microprofile.tooling.server.configuration.ConfigurationException;
+import org.jboss.eap.qe.microprofile.tooling.server.configuration.arquillian.ArquillianContainerProperties;
+import org.jboss.eap.qe.microprofile.tooling.server.configuration.arquillian.ArquillianDescriptorWrapper;
+
+public class HealthUrlProvider {
```

Comment:
Could you move it to subpackage, something like `tooling`|`helper`, please?

Reply:
Done